build.sbt

```diff
@@ -58,7 +58,7 @@ lazy val commonSettings =
     unmanagedSourceDirectories.in(Compile) := Seq(scalaSource.in(Compile).value),
     unmanagedSourceDirectories.in(Test) := Seq(scalaSource.in(Test).value),
     publishMavenStyle := false,
-    crossSbtVersions := Seq("1.1.6", "0.13.17")
+    crossSbtVersions := Seq("1.1.6")
```

Comment:
Should this include the latest and greatest, i.e. 1.3.3? Do we even need cross building (is 1.x compatible)?

Reply:
My understanding is that 1.x is compatible, I was just lazy leaving a single cross version to not have to also update the travis file 😬